pulp_smash/tests/docker/__init__.py

```diff
@@ -0,0 +1 @@
+"""This package contains tests for the Docker plugin."""
```

Comment:
Same nitpick. I start every module with an encoding declaration and unicode literals, whether needed or not.

```python
# coding=utf8
"""This package contains tests for the Docker plugin."""
from __future__ import unicode_literals
```